### ISSUE TITLE
Grip Issues on Kunckles Controller in Robo Recall.

### ISSUE THREAD

**Issue by DiabeticAnna** [LibreVR/Revive#1618]
**Description of the Issue**

There exists an issue in Robo Recall wherein the controllers stay locked in a permanent "gripped state" on startup, regardless of controller bindings and button presses. Squeezing (gripping) on the controllers registers as the X and A button on the respective controller. 

**Technical Information**

    CPU: AMD Threadripper 2950X
    GPU: Nvidia Geforce RTX 2080
    RAM: 32GB DDR4
    OS: Windows 10
    Valve Index full kit HMD/Kunckles Controllers on latest firmware.

**Versions and Features**

    Revive: 2.1.1.0

**What apps/games are you trying to run?**

    Robo Recall

**How to reproduce your issue?**

    Open up the game (Robo Recall)
    Play through the first mission.
    Try grabbing the robot handle. 

**Error messages/codes?**

    None.

**Comment by Martynas-Mileikis** [comment]
I have the same issue. My grip is registering as an A button

**Comment by sheepdestroyer** [comment]
Same here

**Comment by abuchris** [comment]
@sheepdestroyer Did it fix the "grip registering as A button" issue for you? And if so, how do I grab the beta to test for myself, as that is my issue too.

**Comment by sheepdestroyer** [comment]
> @sheepdestroyer Did it fix the "grip registering as A button" issue for you? And if so, how do I grab the beta to test for myself, as that is my issue too.

yes, grip issue is fixed by beta.
As indicated on readme.md, you can get it from : https://ci.appveyor.com/project/librevr/revive/build/artifacts

However I have the 360° Tracking bug where your waist is at ground level. it seemed to have been fixed a while ago but came back.

**Comment by abuchris** [comment]
Thank you kindly 👍

**Comment by Martynas-Mileikis** [comment]
@abuchris did it work for you?
I've tried the beta, but grip buttons still register as X and A.

**Comment by abuchris** [comment]
Nope, I have the same issue, grips register as X on left, A on right. Also tried it on the beta.

**Comment by will-quast** [comment]
In RoboRecall Win64_4.16.0.0 with **Revive 2.1.1** the Index grip maps to the X and A buttons on the Oculus Touch, this is not expected. When I downgrade to **Revive 1.7.2** with all other settings the same, the Index grip maps to the hand triggers (grips) on the Oculus Touch, this works as expected in game.

**Comment by 17108730** [comment]
I have the same issue. My grip is registering as an A button

**Comment by Martynas-Mileikis** [comment]
> I have the same issue. My grip is registering as an A button

Buy a legal copy of the game. I had this issue only with a cracked version
